app/javascript/shared/components/tags.jsx

```diff
@@ -321,20 +313,24 @@ class Tags extends Component {
       });
     }
     const { listing } = this.props;
-    return this.index
-      .search(query, {
-        hitsPerPage: 8,
-        attributesToHighlight: [],
-        filters: 'supported:true',
-      })
-      .then(content => {
+    return fetch(`/tags/search?name=${query}`, {
+      method: 'GET',
+      headers: {
```

Comment:
Not for this PR, but I imagine we reuse these same headers for almost all calls on the frontend for our endpoints. It might be good to consider having this consolidated so a dev doesn't need to add common headers every time a `fetch` call is made.